src/main/java/com/minecolonies/coremod/colony/requestsystem/requests/StandardRequestFactories.java

```diff
@@ -277,15 +274,14 @@ public NBTTagCompound serialize(@NotNull final IFactoryController controller, @N
         }
     }
 
-    @SuppressWarnings(Suppression.BIG_CLASS)
     public static final class BurnableRequestFactory implements IRequestFactory<Burnable, StandardRequests.BurnableRequest>
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce this class from 41 to the maximum allowed 30 or externalize it in a public class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS2972)